src/test/resources/features/ManagementUI.feature

```diff
@@ -0,0 +1,66 @@
+Feature: User manages HOCS teams and units
+
+  Background:
+    Given that I have navigated to the Management UI as the user "EAMON"
+
+  Scenario: User navigates to a management page
+    When I navigate to the "STANDARD LINE" Management page
+    Then I should be taken to the "STANDARD LINE" Management page
+
+  @HOCS-832
+  Scenario: User can not see any assigned users if team does not have any
+    And I navigate to the "TEAM" Management page
+    When I search for a team with no assigned users
+    Then no users should be shown in user list
+
+  @HOCS-832
+  Scenario: Adding a new user to a team displays that user in the team list
+    And I navigate to the "TEAM" Management page
+    When I select the "Pursue Disruptions Unit" team from the dropdown
+    And I add the user "eamon.droko@ten10.com" to the team
+    Then the user should be visible in the team list
+
+  Scenario: User can add multiple users to a team
+    And I navigate to the "TEAM" Management page
+    When I select the "OSCT Secretariat" team from the dropdown
+    And I add the users "eamon.droko@ten10.com" and "danny.large@ten10.com" to the team
+    Then the users should visible in the team list
```

Comment:
Cant see where we delete the users after to reset test to initial state?

Reply:
I don't actually have that. May need to add that functionality